domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/statementofmeans/Employment.java

```diff
@@ -0,0 +1,73 @@
+package uk.gov.hmcts.cmc.domain.models.statementofmeans;
+
+import lombok.Builder;
+import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
+import uk.gov.hmcts.cmc.domain.constraints.EachNotNull;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import javax.validation.Valid;
+
+import static java.util.Collections.emptyList;
+import static uk.gov.hmcts.cmc.domain.utils.ToStringStyle.ourStyle;
+
+@Builder
+public class Employment {
+
+    @Valid
+    @EachNotNull
+    private final List<Employer> employers;
+
+    @Valid
+    private final SelfEmployment selfEmployment;
+
+    @Valid
+    private final UnEmployment unEmployment;
```

Comment:
Can we make it `Unemployment` not `UnEmployment `? I think we should not capitalise E.